# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cache/DBSchema.cpp
# Commit: 98a751f6f41d
# Full Hash: 98a751f6f41d4e8d23f38aa0aa1d483f83cd0bda
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2023-01-19 04:46:07
# Regressor Bug: 1247687
# File Overlap Count: 1
# Description:
#   Bug 1247687 - Implement csp for Module Workers; r=evilpie,asuth,rpl,ckerschb
#   
#   Depends on D155691
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156102
# ==============================================================================

diff -r 78addb6af24d -r 98a751f6f41d dom/cache/DBSchema.cpp
--- a/dom/cache/DBSchema.cpp	Wed Jan 18 13:46:32 2023 +0000
+++ b/dom/cache/DBSchema.cpp	Wed Jan 18 13:46:33 2023 +0000
@@ -342,7 +342,9 @@
         nsIContentPolicy::TYPE_INTERNAL_FRAME_MESSAGEMANAGER_SCRIPT == 53 &&
         nsIContentPolicy::TYPE_INTERNAL_FETCH_PRELOAD == 54 &&
         nsIContentPolicy::TYPE_UA_FONT == 55 &&
-        nsIContentPolicy::TYPE_WEB_IDENTITY == 57,
+        nsIContentPolicy::TYPE_WEB_IDENTITY == 57 &&
+        nsIContentPolicy::TYPE_INTERNAL_WORKER_STATIC_MODULE == 58 &&
+        nsIContentPolicy::TYPE_END == 59,
     "nsContentPolicyType values are as expected");
 
 namespace {